content/member-handbook.md

```diff
@@ -147,6 +147,186 @@ That said, here are some things we might do, in rough order of severity:
   Usually used when someone refuses to step back from a heated conversation or for trolls.
 - **ban**:  asking someone to log off completely or doing the same via admin tools.
 
+## Job Postings
+
+The `#jobs` channel is a space that we've dedicated for people to post when they
+are looking for work, or when they have jobs.  This section will provide some
+guidance and rules for posting in the `#jobs` channel.
+
+### Pinning Posting Durations
+
+#### About Pinning
+
+Slack allows posts to be pinned to a channel.  This provides a way for job
+openings and for-hire posts to be kept around and highlighted in a channel after
+the conversation has moved on.
+
+Due to the restrictions placed on us by being a free Slack team, the full
+contents of pinned posts may not be available after 10,000 messages have been
+posted (historically that's meant 2-3 weeks for our team).  To work around this
+limitation we strongly encourage you to use a very short summary at the top of
+any posting, so that it's useful when pinned afer the full message has expired.
+
+An example of using this strategy would be:
+
+```
+Data Scientist IV at Rackspace (DM: @rebecca.skinner) https://goo.gl/abcd
+
+<rest of the message>
+```
+
+#### Pinned Message Durations
+
+Pinned messages more than 90 days old will be periodically cleared out by
+administrators.  If you have fulfilled the pinned request (filled the role,
+gotten a job) then we would request that you unpin the message yourself.
+
+#### Pinned Message Eligiability
+
+A post may be pinned if there is no similar post that has already been pinned.
+That is to say, if you posted a job listing last week and it was pinned, then
+you post again today, you need to unpin the previous listing if you want to pin
+the current one.  No hard restrictions are placed on how often you may unpin old
+posts and re-pin new ones, although moderators may request that you reduce
+the frequency of pinning our our discression if it appears to be interfering
+with the normal functioning of the `#jobs` channel.
+
+In order to encourage maximum transparency in our job postings, we place more
+stringent requirements for pinned job postings than unpinned postings.
+Aggregate job listings are not eligible for pinning.  Furthermore, a pinned job
```

Comment:
The talk about "eligibility" makes this all sound very formal, when this is really just setting some expectations around a loose community practice that already exists. The tone could be more peer-to-peer than grantor-to-grantee.